singularity-opac/photons/mean_opacity_photons.hpp

```diff
@@ -186,8 +186,8 @@ class MeanOpacity {
   PORTABLE_INLINE_FUNCTION Real fromLog_(const Real lx) const {
     return std::pow(10., lx);
   }
-  Spiner::DataBox lkappaPlanck_;
-  Spiner::DataBox lkappaRosseland_;
+  Spiner::DataBox<Real> lkappaPlanck_;
```

Comment:
Thanks for fixing the `mean_s_opacity` functions as well -- should we move these original changes to the `using Databox = ` pattern you have elsewhere for consistency? I can do this if you want to hand the PR back to me

Reply:
Sure, that sounds good. Thanks!